waku/node/peer_manager/peer_manager.nim

```diff
@@ -10,6 +10,7 @@ import
   chronicles,
   metrics,
   libp2p/multistream,
+  libp2p/connmanager,
```

Comment:
Is this really needed?

Reply:
good catch, fixed
47e3a4942cbf2ed42d41b2093e99733e2329dde0